blocks/color-palette/style.scss

```diff
@@ -22,6 +22,12 @@ $color-palette-circle-spacing: 14px;
 	&:hover {
 		transform: scale( 1.2 );
 	}
+
+	// Ensure that the <div> that <Dropdown> wraps our toggle button with is full height
```

Comment:
Hmm, considering the `div` was added as intentionally expected to be unstyled, adding styles here seems to defeat its purpose to an extent. See #3958.

Reply:
Yeah, fair. 

From https://github.com/WordPress/gutenberg/pull/3958#issuecomment-355099629:

> An alternative approach I considered was to account for / ignore padding in the popover positioning, but this would be non-trivial to implement because the popover component allows for custom behavior to define anchor parent bounds.

Is this something we should re-consider?

Reply:
Let's review this approach after the release. It doesn't seem necessary for this case either, so adding redundant divs should be avoided.